### PR TITLE
Update diagnostic messages as of 3/23/22

### PR DESCRIPTION
**Pull request by parlough**
I staged these changes locally and they all read and look great.

Closes https://github.com/dart-lang/site-www/issues/3945

### CODE REVIEW

**Review by bwilkerson — APPROVED** [review]
Shams already reviewed the text before I committed it to the SDK. I don't know whether she wants to look it over again before we merge this PR.

(edit)
Ok, she needs to because I don't have write access.

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**
